front/src/modules/ui/field/hooks/useIsFieldEmpty.ts

```diff
@@ -0,0 +1,13 @@
+import { useContext } from 'react';
+import { useRecoilState } from 'recoil';
+
+import { FieldContext } from '../contexts/FieldContext';
+import { isFieldEmptyScopedState } from '../states/isFieldEmptyScopedState';
+
+export const useIsFieldEmpty = () => {
```

Comment:
This needs to be improved because it would require every field to set this state, maybe we can find a way to check if a field is empty by directly checking the field value ?